pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -206,10 +208,7 @@ func (exp *flowExporter) sendTemplateRecord(templateRec ipfix.IPFIXRecord) (int,
 		}
 	}
 	for _, ie := range IANAReverseInfoElements {
-		split := strings.Split(ie, "_")
-		runeStr := []rune(split[1])
-		runeStr[0] = unicode.ToLower(runeStr[0])
-		element, err := exp.process.GetIANARegistryInfoElement(string(runeStr), true)
+		element, err := exp.registry.GetInfoElement(ie, ipfixregistry.ReverseEnterpriseID)
```

Comment:
what is `ReverseEnterpriseID` and why is it needed in addition to `IANAEnterpriseID`?

Reply:
Ok you can ignore this comment, I got confused by the naming. In the future, I would suggest using `RevereInformationElementEnterpriseID`

Reply:
Sounds good. `ReverseEnterpriseID` is the enterprise ID for registry with reversed information elements from IANA registry.